sql/moz-fx-data-shared-prod/search_terms_derived/adm_daily_aggregates_v1/metadata.yaml

```diff
@@ -7,7 +7,7 @@ description: |-
   The sending to adMarketplace takes place in a separate DAG defined in
   the telemetry-airflow repository directly.
 owners:
-  - jklukas@mozilla.com
+  - ctroy@mozilla.com
```

Comment:
Just to make sure I understand it all: what is/happens in this job?

Reply:
This query populates a table of daily count of engagement per allowlisted search term. There's a separate DAG defined in telemetry-airflow that sends the contents of this table to a partner SFTP server: https://github.com/mozilla/telemetry-airflow/blob/main/dags/adm_export.py